### PR TITLE
Editorial alignment with JOSE and HTTP terminology.

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

- use JOSE header parameter
- HTTP header field / field value
- simplify terminology

### CODE REVIEW

**Comment by ioggstream** [comment]
@b---c if you agree with this PR content, I can rebase it. Let me know, R.

**Comment by ioggstream** [comment]
@b---c if you agree with this PR content, I can rebase it. Let me know, R.

File: main.md [hunk 1]
```diff
@@ -379,38 +379,32 @@ are included for formatting and readability.
...
-The idea is sign just enough of the HTTP data to
-provide reasonable proof-of-possession with respect to the HTTP request. But 
-that it be a minimal subset of the HTTP data so as to avoid the substantial 
-difficulties inherent in attempting to normalize HTTP messages. 
-Nonetheless, DPoP proofs can be extended to contain other information of the
-HTTP request (see also (#request_integrity)).
+DPOP JWT conveys only the HTTP method and the request target.
+This is to provide a reasonable correlation with the HTTP request. 
+Nonetheless, DPoP proofs can be extended to contain other information
+related to the HTTP request (see also (#request_integrity)).
```
Comment (selfissued) [inline main.md:385]: I disagree with the deletion of the text above about the two headers of the request, etc.  The replacement text does not equivalently convey the information that would have been deleted.

File: main.md [hunk 2]
```diff
@@ -379,38 +379,32 @@ are included for formatting and readability.
...
-    request in which the JWT was received,
- 1. the `htu` claim matches the HTTPS URI value for the HTTP
-    request in which the JWT was received, ignoring any query and
-    fragment parts,
+    JOSE header parameter,
+ 1. the `jwk` JOSE header parameter does not contain a private key,
+ 1. the `htm` claim matches the HTTP method of the
+    current request,
+ 1. the `htu` claim matches the current request target,
+    ignoring any query and
```
Comment (selfissued) [inline main.md:406]: I don't like that the explanatory text "HTTPS URI value for the HTTP request in which the JWT was received" is lost in the rewrite.  The replacement text "current request target" could easily be misunderstood by readers.

**Review by selfissued — CHANGES_REQUESTED** [review]
This PR mostly contains useful wording improvements.  That said, there are a few places where the new wording loses information present in the original text that would be useful to readers.

Please revise to retain this information.  Thanks.

File: main.md (hunk 1 above)
Comment (b---c) [reply main.md:385]: Yes, please restore the original paragraph of text.

File: main.md [hunk 3]
```diff
@@ -335,11 +335,11 @@ The payload of a DPoP proof contains at least the following claims:
    pseudorandom data or by using a version 4 UUID string according to [@RFC4122].
    The `jti` can be used by the server for replay
    detection and prevention, see (#Token_Replay).
- * `htm`: The HTTP method for the request to which the JWT is
+ * `htm`: The HTTP method of the request to which the JWT is
    attached, as defined in [@!RFC7231] (REQUIRED).
- * `htu`: The HTTP URI used for the request, without query and
+ * `htu`: The request URI {{Section 5.5 of RFC7230}}, without query and
```
Comment (b---c) [inline main.md:340]: this markdown is written for mmark not kramdown so that syntax doesn't work

Comment (b---c) [inline main.md:340]: this markdown is written for mmark not kramdown so that syntax doesn't work

File: main.md (hunk 1 above)
Comment (ioggstream) [reply main.md:385]: Pls see:

- https://github.com/danielfett/draft-dpop/pull/146/files#r855935610
- https://github.com/danielfett/draft-dpop/pull/146/files#r850816046

File: main.md (hunk 2 above)
Comment (ioggstream) [reply main.md:406]: Ok, see https://github.com/danielfett/draft-dpop/pull/146/files#r855935610

**Comment by ioggstream** [comment]
Comments moved to #146

**Pull request closed without merging**